docs/features/F003-3d-visualization/plan.md

```
@@ -0,0 +1,103 @@
+
+| 구분 | 선택 | 이유 |
+| --- | --- | --- |
+| (영역) | (기술) | (선택 이유) |
+
+---
+
+## 아키텍처
+
+(컴포넌트 구조, 데이터 흐름 등)
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)